client/webserver/site/src/js/wallets.ts

```diff
@@ -1521,7 +1522,7 @@ export default class WalletsPage extends BasePage {
     const { page, selectedAssetID: assetID, mixerStatus } = this
     if (!mixerStatus) return
     Doc.hide(page.mixConfigErr)
-    const cfg = { assetID: assetID } as any
+    const cfg = { assetID: assetID, IsMixingEnabled: page.enableMixing } as any
     const loaded = app().loading(page.mixerConfigForm)
     if (Doc.isDisplayed(page.mixAddrInputBox)) {
       cfg.serverAddr = page.mixAddrInput.value
```

Comment:
Do users still need to provide mixing server addr? Can remove form elements that are no longer required.

Reply:
No they don't It is no longer required. I will remove it.

Reply:
Okay.